README.md

```diff
@@ -71,6 +74,45 @@ We have put some coarse mitigations into place to try to protect against this
 situation, but it's still not good practice to do it in the first place.  See
 https://github.com/vector-im/element-web/issues/1977 for more details.
 
+Configuration best practices
+----------------------------
+
+Unless you have special requirements, you will want to add the following to
+your web server configuration when hosting Element Web:
+
+- The `X-Frame-Options: SAMEORIGIN` header, to prevent Element Web from being
+  framed and protect from [clickjacking][owasp-clickjacking].
+- The `frame-ancestors 'none'` directive to your `Content-Security-Policy`
+  header, as the modern replacement for `X-Frame-Options` (though both should be
+  included since not all browsers support it yet, see
+  [this][owasp-clickjacking-csp]).
```

Comment:
I don't think this is needed given the embedded CSP saying `default-src 'none';`

https://github.com/vector-im/element-web/blob/develop/src/vector/index.html#L27

Reply:
The `default-src` directive [does not](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/default-src) set the default for `frame-ancestors`. Perhaps you were thinking of `frame-src`?

This is to prevent other pages framing Element.

Reply:
Good to know, I tried to find this in the docs and read the opposite between the lines